BudgetTests/DataControllerTests.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  DataControllerTests.swift
+//  BudgetTests
+//
+//  Created by hoang on 06.01.2022.
+//
+
+import XCTest
+import CoreData
+
+struct DataControllerTests {
```

Comment:
This file seems to be the tests file (as it imports XCTest and have Tests postfix) but actually having no tests, feels strange. I think a renaming might be needed, my suggestion - DataControllerTestImpl